lib/context_adapter_server.js

```diff
@@ -231,19 +222,73 @@ function onThirdPartyResponse(request, err, response) {
  * @param {Object} request The request received
  * @param {Object} serviceDescriptor The service descriptor associated to the received request
  */
-function processOperationRequest(request, serviceDescriptor) {
-  // Valid service description retrieved
-  caLogger.debug(
-    request.contextAdapter.context,
-    'service descriptor: ' +
-      JSON.stringify(serviceDescriptor)
-  );
-
+function processOperationRequest(request) {
   // Send a request to the Third Party
   caComm.sendThirdPartyRequest(request, onThirdPartyResponse.bind(null, request));
 }
 
 /**
+ * Handler of geolocation service responses
+ * @param {Object} request The received request
+ * @param {Object} cellIdDescriptor The cell id descriptor whose geolocation has been retrieved
+ * @param {Object} err Error, if any, when sending the request to the third party
+ * @param {Object} response The response, if any, received from the third party
+ */
+function onGeolocationResponse(request, cellIdDescriptor, err, response) {
+  if (err || response.body.error) {
+    // Error response from the geolocation service
+    caLogger.warn(
+      request.contextAdapter.context,
+      'Error response from the geolocation service: ' +
+      (err ? err.code + ' - ' + err.message : JSON.stringify(response.body.error))
+    );
+    caLogger.warn(
```

Comment:
Just for readability and because the second one was initially set as debug but I raised it to warn just to notify about the issue... :)